### DISCUSSION TITLE
What's the correct way of updating?

### DISCUSSION THREAD

**Discussion by ariasmn** [AdnanHodzic/auto-cpufreq#340] [Q&A] [answered]
In case we installed it using the `auto-cpufreq-installer`, what's the correct way of updating the app? Just download the new release and run the installer again?

Thanks for the amazing work, by the way!

**Comment by AdnanHodzic** [comment]
> Thanks for the amazing work, by the way!

Most welcome!

> In case we installed it using the `auto-cpufreq-installer`, what's the correct way of updating the app? Just download the new release and run the installer again?

Click on "Notifications" button as top of auto-cpufreq repo and subscribe to get notifications you're interested in, whether those are only "releases" or something else is up to you.

**Reply by ariasmn** [reply]
Thank's for the heads up!

But what I really meant is: how can I update the application that I already installed in my computer? For example, right now I'm running the v1.7.2 version, and I want to update to the latest release (v1.9.0). What's the correct way to do so? Uninstall the current version before installing the new one? Just run the installer of the new release?

Don't know if I'm explaining myself, sorry!

**Comment by AdnanHodzic** [comment, marked as answer]
Right, you can uninstall just to make sure, but 

```
git pull
sudo ./auto-cpufreq-installer --install
sudo auto-cpufreq --install
```

Should be enough.

**Reply by ariasmn** [reply]
Nice!

Would be possible to implement a command to automatically update? Something like `sudo auto-cpufreq --update`, that checks if the actual version is the latest and, if not, do the whole updating process?

Don't know if that makes sense or even if it's possible at all.

**Reply by AdnanHodzic** [reply]
Sure, create an issue and add `[request]` prefix to its title. 

I probably won't have time to look into this right now, but if you want to give it a try I would definitely invite you to [contribute ](https://github.com/AdnanHodzic/auto-cpufreq#code-contribution)this feature yourself and get credit for it :)
